lib/IR/AsmWriter.cpp

```diff
@@ -3366,6 +3366,32 @@ void MDNode::printAsBody(raw_ostream &OS, ModuleSlotTracker &MST, const Module *
     TypePrinter.incorporateTypes(*M);
   WriteMDNodeBodyInternal(OS, this, &TypePrinter, MST.getMachine(), M);
 }
+
+#if defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)
```

Comment:
Sorry, I put this wrong in my comment, it should be `#if !defined(NDEBUG) || defined(LLVM_ENABLE_DUMP)`